src/prefect/cli/project.py

```diff
@@ -87,6 +103,46 @@ async def init(
         key, value = field.split("=")
         inputs[key] = value
 
+    if not recipe:
+        if confirm(
+            (
+                "You haven't specified a recipe. Would you like to see a list of"
+                " available recipes?"
+            ),
+            default=True,
+            console=app.console,
+        ):
```

Comment:
I think we can take the message for this `confirm` and use it as the message for the `prompt_select_from_table` to reduce the number of prompts shown to the user.

Reply:
Good idea